app/components/polaris/popover_component.html.erb

```diff
@@ -7,25 +7,27 @@
     <% end %>
   </div>
 
-  <%= render Polaris::BaseComponent.new(**system_arguments) do %>
-    <%= render Polaris::BaseComponent.new(**popover_arguments) do %>
-      <div class="Polaris-Popover__FocusTracker" tabindex="0"></div>
-      <div style="">
-        <div class="Polaris-Popover__Wrapper">
-          <%= render Polaris::BaseComponent.new(**content_arguments) do %>
-            <% if panes.present? %>
-              <% panes.each do |pane| %>
-                <%= pane %>
-              <% end %>
-            <% else %>
-              <%= render Polaris::Popover::PaneComponent.new(sectioned: @sectioned) do %>
-                <%= content %>
+  <div data-polaris-popover-target="template" style="display: none;">
```

Comment:
Initially I was trying to use a `<template>` tag here but capybara removes all the content from inside a template tag to mimic the browser which made it harder to test this component.

See: https://github.com/teamcapybara/capybara/issues/2510

Reply:
Yeah, `template` tag seems to have more sense in that case. I guess we could avoid this issue by using system tests for popover-based components.

Reply:
That would be perfect yes.